bin/merge_selection_metrics.py

```diff
@@ -19,7 +19,8 @@ def seqmeta_reader(filename):
 
 def selection_metrics_reader(filename):
     with open(filename) as fh:
-        return json.load(fh)['events'][0]['tree-info']['lb']
+        events = json.load(fh)['events'][0]
```

Comment:
@metasoarous Do we want to leave this as using `events[0]`? It's a separate issue from the changes I am making here but wanted to raise the question, since in process_partis.py it seems like you find the appropriate cluster annotation among `events`, but maybe index 0 is taken here because these annotations are the result of [the special partis annotate command for selection metrics](https://github.com/matsengrp/cft/blob/master/SConstruct#L716)

Reply:
I don't know enough of the context here around when you might get multiple events and have to worry about this, but I _think_  you're right that in this particular case we expect to always only have one event. Maybe we should check with @psathyrella?

Reply:
Well just in principle, it pains my soul to use an index on a list whose length I'm not verifying. That aside, I would strongly recommend using the existing functions in partis for reading partis output files, like you're doing in [process_partis.py](https://github.com/matsengrp/cft/blob/master/bin/process_partis.py#L215), you'll be much less likely to run into problems.

That doesn't tell you which event to take, though -- that's between what command was used to create the file and what you want here. It would be best to choose the event with the right uids that were  passed in, to enforce correspondence between the previous calling line and this line reading the output. But it would also be fairly safe to do something like


```
if len(annotation_list) != 1:
    raise Exception('expected one event from %s but got %d' % (fname, len(annotation_list)))
event = annotation_list[0]
```

Reply:
@metasoarous Should I create a new issue for fixing this if you agree with Duncan?

Reply:
Yup; Sounds good to me.